src/transformers/models/llama4/configuration_llama4.py

```diff
@@ -335,11 +340,15 @@ def __init__(
         self.output_router_logits = output_router_logits
         self.router_aux_loss_coef = router_aux_loss_coef
         self.router_jitter_noise = router_jitter_noise
+
+        # Backwards compatibility
+        if no_rope_layers == []:
+            no_rope_layers = None
+
         default_no_rope_layers = [
             int((layer_idx + 1) % no_rope_layer_interval != 0) for layer_idx in range(self.num_hidden_layers)
         ]
```

Comment:
Reverted to the previous, exhaustive list. Note that `1` means `RoPE`, while `0` means `NoPE`. But the property is called `no_rope_layers`, so it may be confusing. Let me know if you want me to use `1`s for NoPE instead @ArthurZucker.

Reply:
it would be breaking I think no? Otherwise the cache needs an update as well